File: pgm/ABC_Java/ABC_262_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final int Y = Integer.parseInt (reader.readLine());
        final int two = 2;
        final int four = 4;
        final int mod = Y % four;
        int year = Y + (four - two) - mod;
        if (mod > two) {
            year += four;
        }
        System.out.println (year);
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_262_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.*
// import java.util.Scanner
// import java.util.StringTokenizer

// import kotlin.math.*

const val space: String = " "
const val two: Int = 2
const val four: Int = 4
const val zero: Int = 0

fun main() {
    val Y: Int = readln().toInt()
    val target: Int = Y + (four - two) - Y % four + if (Y % four > two) {
        four
    } else {
        zero
    }
    println(target)
    kotlin.system.exitProcess(0)
}